internal/map.go

```diff
@@ -539,29 +539,30 @@ func (mp *mapProxy) AddEntryListenerToKeyWithPredicate(listener interface{}, pre
 func (mp *mapProxy) onEntryEvent(keyData *serialization.Data, oldValueData *serialization.Data,
 	valueData *serialization.Data, mergingValueData *serialization.Data, eventType int32, uuid *string,
 	numberOfAffectedEntries int32, includedValue bool, listener interface{}) {
+	member := mp.client.ClusterService.GetMemberByUUID(*uuid)
 	key, _ := mp.toObject(keyData)
 	oldValue, _ := mp.toObject(oldValueData)
 	value, _ := mp.toObject(valueData)
 	mergingValue, _ := mp.toObject(mergingValueData)
-	entryEvent := protocol.NewEntryEvent(key, oldValue, value, mergingValue, eventType, uuid)
-	mapEvent := protocol.NewMapEvent(eventType, uuid, numberOfAffectedEntries)
+	entryEvent := protocol.NewEntryEvent(mp.name, member, eventType, key, oldValue, value, mergingValue)
```

Comment:
Why are we changing this order?

Reply:
Here, `member` and `name` were missing. I added them.